### ISSUE TITLE
Cursor creates text selections without dragging the mouse

### ISSUE THREAD

**Issue by n-heartstrings-sterling** [rubberduck-vba/Rubberduck#5873]
**Rubberduck version information**
	Rubberduck version 2.5.2.5906 loading:
	Operating System: Microsoft Windows NT 10.0.19043.0 x64
	Host Product: Microsoft Office x64
	Host Version: 16.0.14530.20000
	Host Executable: EXCEL.EXE;

**Description**
After (and only after) refreshing using Rubberduck, every second click will inconsistently act as if the mouse is being held down, causing a selection to expand as the mouse moves. On the second click, the selection will remain, A subsequent click will cause a selection to be made towards the cursor, and so on. This only seems to occur in complex user forms.

**To Reproduce**
1. Create a new workbook
2. Add a userform
3. Fill it with a about 100 controls - frames, llistboxes, buttons, multipages,,, ![image](https://user-images.githubusercontent.com/84310111/136295947-554f39a4-107c-4ecf-8786-ce7eb166c98f.png)
4. Handle ten or so click events
5. Fill each handler with thousands of lines of junk. My original file had about 1710 lines of proper code, but I managed to reproduce it using 11,000 lines of ```For i = 1 To 1
   Stop
Next``` instead
6. Compile
7. Refresh using Rubberduck
8. Start clicking about in the code, moving your mouse down and up each time, never holding down click, refreshing again if nothing happens ( It seems pretty inconsistent in this test case, but it's horribly consistent in my original case)
9. The cursor should occasionally act as if you are holding down the mouse button, snapping the selection to wherever your mouse moves despite no input

**Expected behavior**
The cursor should move without making any selections

**Screenshots**
![image](https://user-images.githubusercontent.com/84310111/136295864-dac66947-d94e-4f89-a468-60d804181225.png)
![image](https://user-images.githubusercontent.com/84310111/136295853-afc39a78-92b2-491d-b4e6-f6cf550f4851.png)
You'll have to take my word for it - at no point during this recording am I holding down the mouse
https://user-images.githubusercontent.com/84310111/136296711-4ad6a2db-d87c-42ae-ad5e-80bf69644618.mp4

**Logfile**
Opened existing file with above setup, opened the VBE, refreshed and enabled code explorer, opened userform code, refreshed and clicked around until it occurred once, continued until it occurred a second time, closed file.
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/7297667/RubberduckLog.txt)

**Additional context**
This is a fresh install of 21H1 and 365 Beta from ODT. I was previously using Windows 10 LTSC with 365 Current without issue.
(nb: VBE colors are changed using regedit and changing the palette values in VB7.dll with a hex editor)

**Comment by n-heartstrings-sterling** [comment]
If I was to take a guess, the issue exists at lines 2337 and 2354 of the log file 
`2021-10-07 10:29:19.7671;TRACE-2.5.2.5906;Rubberduck.UI.Command.MenuItems.CommandBars.AppCommandBarBase;CancellationTokenSource was already disposed for OnSelectionChange.;`

**Comment by retailcoder** [comment]
Hm, not sure how that could happen.. these log entries stem from a cancelled attempt (could be because another event came in by the time the handler entered) to update the commandbar labels and commands accordingly with the current selection changing, captured by our `OnSelectionChange` event... *in response to* the current selection changing.
We do have navigation commands that can alter the current selection in the editor, but none of it seems to be at play here. So that's... interesting.

**Comment by n-heartstrings-sterling** [comment]
Yes, as I've found, turning off autocompletion features didn't seem to fix the issue. It's pretty clear that the mouse clicking issue is _some_ kind of lag-induced loss of the mouseUp event through a race condition or similar, but I'm not sure what or how. It's very possible that this issue would occur in an unmodified VBE given the right latency at the right time

**Comment by lvangh** [comment]
Just chiming in - seeing the same issue - I can't offer specifics about the environment since I restarted Excel, but I had 3 workbooks open, 2 add-ins, and a large userform project open with hundreds of control objects etc. I was in the RD menu bar looking at something in the refactor menu, then went to the code section and it started selecting blocks when I clicked. Not sure what else to offer. Using Excel 365 latest version.
